core/src/test/java/org/elasticsearch/ElasticsearchExceptionTests.java

```diff
@@ -516,6 +518,41 @@ public void testFromXContentWithHeadersAndMetadata() throws IOException {
         assertThat(cause.getMetadata("es.index_uuid"), hasItem("_na_"));
     }
 
+    public void testThrowableToAndFromXContent() throws IOException {
+        final XContent xContent = randomFrom(XContentType.values()).xContent();
+
+        final Tuple<Throwable, ElasticsearchException> exceptions = randomExceptions();
+        final Throwable throwable = exceptions.v1();
+
+        BytesReference throwableBytes = XContentHelper.toXContent((b, p) -> {
+            ElasticsearchException.generateThrowableXContent(b, ToXContent.EMPTY_PARAMS, throwable);
```

Comment:
we may have done this somewhere else but I think for correct-ness we should rather pass p in instead of empty params. I would also love to call them builder and params if possible.